lab_4/styles.css

```diff
@@ -0,0 +1,188 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+}
+
+.top {
+    display: flex;
+    justify-content: center;
+
+}
+
+header {
+    width: 1200px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    min-height: 60px;
+    padding-left: 5%;
+    padding-right: 5%;
+}
+
+#burger > a > img {
+    height: 1.9rem;
+}
+
+#logo > a:link, #logo > a:visited {
+    color: grey;
+    font-size: 1.9rem;
+    text-decoration: none;
+    font-family: Corbel;
+}
+
+.center {
+    display: flex;
+    justify-content: center;
+    background-color: #003ecb;
+}
+
+main {
+    height: 70%;
+    padding: 5% 5% 10% 5%;
```

Comment:
issue(blocking): this relative values would be too different on different screens